packages/mobile/src/navigator/NavigatorWrapper.tsx

```diff
@@ -200,6 +216,26 @@ export const NavigatorWrapper = () => {
           <AlertBanner />
           <InviteFriendModal isVisible={isInviteModalVisible} onInvite={onInvite} />
         </View>
+        <Modal isVisible={shakeForSupportVisible} backdropOpacity={0.5} style={styles.modal}>
+          <View style={styles.shakeForSupport}>
```

Comment:
We should wrap this in `<SafeAreaView edges={['bottom']}>` so it doesn't collide with (or stay too close to) the home indicator on iOS devices.